### PR TITLE
EHN: logical_or added in paddle tesnor.py

### PR DESCRIPTION
**Pull request by MuhammadNizamani**
Logical_or is added in paddle.tesnor.tesnor.Tensor from issue  #17596

### CODE REVIEW

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

File: ivy/functional/frontends/paddle/tensor/tensor.py [hunk 1]
```diff
@@ -209,6 +209,24 @@ def allclose(self, other, rtol=1e-05, atol=1e-08, equal_nan=False, name=None):
...
+                "int32",
+                "int64",
+                "float32",
+                "float64",
+            )
+        },
+        "paddle",
+    )
+    def logical_or(self, y, out=None, name=None):
+        y_ivy = _to_ivy_array(y)
```
Comment (Dsantra92) [inline ivy/functional/frontends/paddle/tensor/tensor.py:227]: You need to wrap the variables explicitly for frontend functions.

**Review by Dsantra92 — CHANGES_REQUESTED** [review]
Thanks for the contribution. Just a simple change and we will be good to merge!!

File: ivy/functional/frontends/paddle/tensor/tensor.py (hunk 1 above)
Comment (MuhammadNizamani) [reply ivy/functional/frontends/paddle/tensor/tensor.py:227]: could please you explain why do I need  wrap the variables explicitly for frontend functions?
 _to_ivy_array(y) convert it into ivy_array

Comment (MuhammadNizamani) [reply ivy/functional/frontends/paddle/tensor/tensor.py:227]: @Dsantra92  are you talking about this way to implemet the function ?

```
def logical_or(self, y, out=None, name=None):
        return paddle_frontend.logical_or(self, y, out=out)
```

**Comment by bipinKrishnan** [comment]
Hey @MuhammadNizamani  🙂, I will link this to the corresponding issue, but in future you should do this yourself by adding a comment ‘Close #Issue_number’, as explained [here](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

**Comment by bipinKrishnan** [comment]
Close #17596

File: ivy/functional/frontends/paddle/tensor/tensor.py (hunk 1 above)
Comment (bipinKrishnan) [reply ivy/functional/frontends/paddle/tensor/tensor.py:227]: > @Dsantra92 are you talking about this way to implemet the function ?
> 
> ```
> def logical_or(self, y, out=None, name=None):
>         return paddle_frontend.logical_or(self, y, out=out)
> ```
Yeah, this is what @Dsantra92 meant probably.

**Comment by MuhammadNizamani** [comment]
@bipinKrishnan ok sir got it

**Comment by bipinKrishnan** [comment]
LGTM! Thanks @MuhammadNizamani :rocket:

**Pull request merged**